.eslintrc.js

```diff
@@ -2,78 +2,88 @@ module.exports = {
   root: true,
   env: {
     node: true,
-    browser: true,
+    browser: true
   },
   extends: [
     // Base extends
-    'eslint:recommended',
-    
+    "eslint:recommended",
+
     // Vue extends
-    'plugin:vue/base',
-    'plugin:vue/essential',
-    'plugin:vue/strongly-recommended',
-    'plugin:vue/recommended',
+    "plugin:vue/base",
+    "plugin:vue/essential",
+    "plugin:vue/strongly-recommended",
+    "plugin:vue/recommended",
+    "prettier"
   ],
```

Comment:
While we're at it, might as well add some files to ignore:
```suggestion
  ],
  "ignorePatterns": [".eslintrc.js", "node_modules/"],
```

Reply:
commented out?

Reply:
yea can't use this it won't let me commit it

`ERROR  Error: ESLint configuration in C:\Users\chris\Documents\Documents\Projects\ONLINKER\event-discovery-prototype\.eslintrc.js is invalid:
        - Unexpected top-level property "ignorePatterns".`